src/plugins/thirdroom/thirdroom.game.ts

```diff
@@ -419,7 +419,6 @@ async function onEnterWorld(ctx: GameState, message: EnterWorldMessage) {
 }
 
 function onExitWorld(ctx: GameState, message: ExitWorldMessage) {
-  disposeWorld(ctx.worldResource);
```

Comment:
`disposeWorld` was being called twice:
https://github.com/matrix-org/thirdroom/pull/548/files#diff-a8af856890bd8b181bb9df732851d6c0c736c601b34186d253d3e9d8e63a2717R498

Reply:
I think we still need to handle the leave world case though. When you click the leave world button.